KevinHardy-Cooper/VApp
# Add support for database migrations & Prepare the database for vulnerability score algorithm

This pull request adds support for database migrations. I have added the necessary database migration scripts to update the database in preparation for the vulnerability score.

To test this out, do the following:
- `git checkout -b db_migrations_support`
- `git pull origin db_migrations_support`
- `npm install`
- `npm install -g db-migrate db-migrate-mysql`
- NOTE: If you get a strange authentication error from the below command, you might need to run the following command in the mysql command line:
- `ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY 'password'`

Now, to run the migrations (NOTE: Prior to this, you will need to have the database.json file setup):
- `db-migrate up`

Check your database an ensure the following:
- You now have a `levels` table with three columns: id, name, amount
- You now have an additional column in the `implications` table: weight
